app/javascript/src/components/Dashboard/Notes/MenuBar.jsx

```diff
@@ -0,0 +1,72 @@
+import React from "react";
+
+import { Search, Plus, Settings } from "neetoicons";
+import { Typography } from "neetoui";
+import { MenuBar as NeetoUIMenuBar } from "neetoui/layouts";
+
+import { TAGS, TYPES, SEGMENTS } from "./constants";
+
+const MenuBar = ({ showMenu }) => (
+  <NeetoUIMenuBar showMenu={showMenu} title="Notes">
+    {TYPES.map((type, idx) => (
+      <NeetoUIMenuBar.Block key={idx} label={type.label} count={type.count} />
+    ))}
+
+    <NeetoUIMenuBar.Block label={"All"} count={10} />
+    <NeetoUIMenuBar.Block label={"Users"} count={15} />
+    <NeetoUIMenuBar.Block label={"Leads"} count={25} />
+    <NeetoUIMenuBar.Block label={"Visitors"} count={30} />
+
+    <NeetoUIMenuBar.SubTitle
+      iconProps={[
+        {
+          icon: Search,
+        },
+      ]}
+    >
+      <Typography
+        component="h4"
+        style="h5"
+        textTransform="uppercase"
+        weight="bold"
+      >
+        Segments
+      </Typography>
+    </NeetoUIMenuBar.SubTitle>
+    {SEGMENTS.map((segment, idx) => (
```

Comment:
Does `segment` have a unique value as part of it? If yes, then please use that over index as the key.

Refer:https://robinpokorny.medium.com/index-as-a-key-is-an-anti-pattern-e0349aece318

Reply:
I have added a new `id` field for the items in the constants file and using it as the `key` now.